docs/articles/documentation/test-api/selecting-page-elements/selector-options.md

```diff
@@ -2,25 +2,54 @@
 layout: docs
 title: Selector Options
 permalink: /documentation/test-api/selecting-page-elements/selector-options.html
-checked: true
+checked: false
 ---
 # Selector Options
 
-You can pass the following options to the [Selector constructor](selectors.md#creating-selectors) and the
-[t.select](selectors.md#one-time-selection) function.
+You can pass the following options to the [Selector constructor](selectors.md#creating-selectors).
 
-* [options.dependencies](#optionsdependencies)
 * [options.boundTestRun](#optionsboundtestrun)
-* [options.text](#optionstext)
-* [options.index](#optionsindex)
 * [options.timeout](#optionstimeout)
 * [options.visibilityCheck](#optionsvisibilitycheck)
+* (***deprecated***) [options.dependencies](#optionsdependencies)
+*  (***deprecated***) [options.text](#optionstext)
+*  (***deprecated***) [options.index](#optionsindex)
+
 
 You can also overwrite the options you have specified before.
 
 * [Overwriting Options](#overwriting-options)
 
+## options.boundTestRun
+
+**Type**: Object
+
+If you need to call a selector from a Node.js callback, assign the current
```

Comment:
To call a selector from a Node.js callback, assign the current [test controller](../test-code-structure.md#test-controller) to the `boundTestRun` option.